.env.production

```
@@ -0,0 +1,23 @@
+
+# Application Settings
+ENVIRONMENT=production
+LOG_LEVEL=INFO
+PORT=8080
+TRACING_ENABLED=false
+SERENA_ENABLED=false
+PUBLIC_BASE_URL=https://project-archangel-api.onrender.com
+
+# Security (generate new ones for production)
+JWT_SECRET_KEY=your-super-secret-jwt-key-change-this-in-production
+ENCRYPTION_KEY=your-32-byte-encryption-key-change-this
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ENCRYPTION_KEY key should go before the JWT_SECRET_KEY key

(UnorderedKey)